ui/ceval/src/ctrl.ts

```diff
@@ -64,6 +64,7 @@ export default function (opts: CevalOpts): CevalCtrl {
   let technology: CevalTechnology = 'asmjs';
   let growableSharedMem = false;
   let supportsNnue = false;
+  const analyzable = opts.analyzable;
```

Comment:
Why is this needed? Can't you just use `opts.analyzable` below? Also, since we so far seem to use British spelling in the code ("analyse"), we should keep it consistent here as well.